contrib/mesos/docs/issues.md

```diff
@@ -1,34 +1,118 @@
 ## Known Issues
 
-### Pod Placement
+This page identifies significant known issues with the Kubernetes-Mesos distribution.
+
+* [General Known Issues](#general-known-issues)
+* [DCOS Package Known Issues](#dcos-package-known-issues), in addendum to the above.
+
+## General Known Issues
+
+These known issues apply to all builds of Kubernetes-Mesos.
+
+### Upgrades
+
+Upgrading your Kubernetes-Mesos cluster is currently "YMMV" status.
+One particularly nasty problem exists with respect to expressing executor (kubelet and kube-proxy) process configuration via command line flags.
+It is **strongly** recommended that all of the Kubernetes-Mesos executors are destroyed before upgrading the Kubernetes-Mesos scheduler component:
+- destroy all daemon controllers running in the cluster, across all namespaces
+- destroy all replication controllers running in the cluster, across all namespaces
+- destroy all pods running in the cluster, across all namespaces
+- invoke the "kamikaze" debug endpoint on the scheduler (e.g. `curl http://10.2.0.5:10251/debug/actions/kamikaze`) to terminate all executors
+
+Not following the above steps prior to upgrading the scheduler can result in a cluster wherein pods fail to launch on some or all nodes - indefinitely.
```

Comment:
In fact they don't fail AFAIK. Don't we decline offers with incompatible executor id group? Then we should never see a TASK_ERROR.